### DISCUSSION TITLE
Supports Binance Web3 Wallet

### DISCUSSION THREAD

**Discussion by Pszz** [family/connectkit#383] [Ideas]
#  Documentation
https://developers.binance.com/docs/binance-w3w/quick-start#integration-with-wagmi-v2

**Comment by lochie** [comment]
https://docs.family.co/connectkit/wallets

**Reply by Pszz** [reply]
But Binance wallet is not an injection wallet, it is similar to WalletConnect

![image](https://github.com/family/connectkit/assets/12093551/f878d12d-8208-4509-bece-56a934a058a3)

When I use Connectkit it doesn't work.
But the way of using `wagmi useConnect` directly will work.


```tsx
import { useConnect } from 'wagmi'

const Demo = () => {
  const { connectors, connect } = useConnect()
     return <>{connectors.map((connector) => (
        <Button
          key={connector.id}
          data-wallet-id={connector.id}
          onClick={() => {
            connect({ connector })
          }}
        >
          {connector.name}
        </Button>
      ))}<>
}
```
